.github/workflows/build.yml

```diff
@@ -93,16 +93,16 @@ jobs:
           ./bootstrap && make build
         working-directory: ./worktree-image
         env:
-          MAKE: make -j2 --output-sync=recurse
-          SAGE_NUM_THREADS: 2
+          MAKE: make -j4 --output-sync=recurse
```

Comment:
Did github changed to 4 cores now?

Reply:
Yes, see https://github.blog/2024-01-17-github-hosted-runners-double-the-power-for-open-source/